### ISSUE TITLE
Image types identified

### ISSUE THREAD

**Issue by Finnberrys** [w3c/wcag3#640]
I'm glad to see this included. In discussions with several blind colleagues, I think this is often missing from alt text. As sighted people, we automatically clock the image type as soon as we see it, and this gives us context for the image itself. A photo of an assistance dog, for example, is very different from an illustration of an assistance dog, even if the content is similar. We often have guidance documents containing screenshots, but if some other image type is suddenly inserted in that document, it can leave a blind person a bit confused unless the alt text begins with the new image type. 

I'm looking forward to seeing how this requirement develops.

**Comment by grace-snow** [comment]
To me, reading "image type" implies a file type, like `jpg`, `png` etc. 

Perhaps "image style" would more clearly communicate the intent of this point? Open to other suggestions of course.
